libass/ass/meson.build

```diff
@@ -0,0 +1,12 @@
+fs = import('fs')
+# This hack is needed because the install directory of the libass public headers
+# ('ass') is different than the name of the directory ('libass') that they live under
+# in the source tree. In the scenario where the libass meson build system is being used
+# as a subproject of some parent project, that parent project will need the libass
+# headers to be copied into the appropriate 'ass' directory either before or during the
+# build process so that they are in an appropriate include path prior to the time that
+# the build system starts trying to build the parent project that needs to include
+# them.
```

Comment:
this should probably still link to the meson issue to make it easy to check up on if a better way is implemented yet